Gemfile

```diff
@@ -0,0 +1,24 @@
+source "https://rubygems.org"
+
+group :test do
+  gem "rake"
+  gem "puppet", :git => 'https://github.com/puppetlabs/puppet.git', :tag => '3.8.0'
+  gem "rspec", '< 3.2.0'
+  gem "rspec-puppet", :git => 'https://github.com/rodjek/rspec-puppet.git'
```

Comment:
copy-pasta, I suspect because my source was keeping up with Puppet 4 changes. I'll switch to following the latest gem.